src/test/java/ezvcard/io/json/JCardSerializerTest.java

```diff
@@ -0,0 +1,159 @@
+package ezvcard.io.json;
+
+import static ezvcard.util.StringUtils.NEWLINE;
+import static org.junit.Assert.assertEquals;
+
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+import com.fasterxml.jackson.core.util.DefaultIndenter;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import ezvcard.VCard;
+
+/**
+ * @author Buddy Gorven
+ */
+public class JCardSerializerTest {
+
+	@Test
+	public void serialize_indented() throws Throwable {
+		VCard vcard1 = new VCard();
+		vcard1.setFormattedName("John Doe");
+
+		VCard vcard2 = new VCard();
+		vcard2.setFormattedName("John Doe");
+
+		ObjectMapper mapper = getMapper();
+
+		final DefaultPrettyPrinter pp = new DefaultPrettyPrinter();
```

Comment:
I haven't had a minute to try it out, but as far as I can tell, you can set a pretty printer on the JsonGenerator, then as long as you don't call any writeRaw methods (currently the only place they are being called is in the indent method), it should all work as expected.

The only issue is that the pretty printer doesn't know what type of object it is serializing until it's done serializing it, so basically your only choices are to add a new line after every array element (as below), or not to indent jcards at all.

I mean, theoretically, it might be possible, when the json generator and pretty printer were both created by RawWriter, to switch out the array indenter depending on what was being serialized, but that may end up depending on jackson implementation details more than is wise. I'll try it, and if it works I'll submit it as a separate pull request, and you can decide on it then.